src/semver/Version.d

```diff
@@ -0,0 +1,139 @@
+/*******************************************************************************
+
+    Utility to simplify working with semantic versioning entities, including
+    parsing and comparison.
+
+*******************************************************************************/
+
+module semver.Version;
+
+/**
+    Describes SemVer aware version instance
+ **/
+struct Version
+{
+    /// major version number
+    int major;
+
+    /// minor version number
+    int minor;
+
+    /// patch version number
+    int patch;
```

Comment:
Let's create separate issue for that. I have created this PR to quickly add something for basic project layout and travis testing :P